<launch>

 <!-- env name="ROSCONSOLE_CONFIG_FILE" value="$(find yjh_bot)/config/roslog.config" /-->
  <!-- <param name="/use_sim_time" value="true" />  -->

<!--   <node name="rosbag" pkg="rosbag" type="play" 
        args="$(find yjh_bot)/bag/data_20200824164300.bag"/>
 -->
  <!-- Load the URDF/Xacro model of our robot -->
  <!-- arg name="urdf_file" default="$(find xacro)/xacro '$(find mybot_description)/urdf/box_robot/box_robot_with_laser.xacro'" / -->
   
  <!-- param name="robot_description" command="$(arg urdf_file)" / -->
    
  <!-- node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0.0 0 0 0  base_footprint base_link 10" /--> 
  <node pkg="tf" type="static_transform_publisher" name="laser_link_broadcaster" args="0.2 0 0.17 0.4 0.5 0.6 base_link laser_link 10" />

  <!-- node pkg="tf" type="static_transform_publisher" name="imu_link_broadcaster" args="0 0 1 0 0 0 base_footprint imu 40" /-->
  <!-- node pkg="tf" type="static_transform_publisher" name="camera_link_broadcaster" args="0.1 -0.03 0.8 0 0 0 base_footprint camera_link 40" /-->
  <!-- node pkg="tf" type="static_transform_publisher" name="sonar_link_broadcaster_0" args="0.1 -0.1 0.32 0 0 0 base_footprint sonar_0 40" /-->
  <!-- node pkg="tf" type="static_transform_publisher" name="sonar_link_broadcaster_1" args="0.1 0.1 0.32 0 0 0 base_footprint sonar_1 40" /--> 

  <!-- robot_pose_ekf -->
<!--   <node name="robot_pose_ekf" pkg="robot_pose_ekf" type="robot_pose_ekf">
      <param name="output_frame"  value="odom" />
      <remap from="odom" to="/odom_cov" />
      <param name="frequency"  value="30.0" />
      <param name="sensor_timeout"  value="1.0" />
      <param name="odom_used"  value="true" />
      <param name="imu_used"  value="true" />
      <param name="debug"  value="false" />
      <param name="self_diagnose"  value="false" />
      <remap from="imu_data" to="imu_data"/>
  </node> -->

  <node name="tf_test_node" pkg="tf_tutorial" type="tf_test_node" output="screen">
    <param name="base_frame" value="base_link" />
  	<param name="child_frame" value="laser_link" />
  	<param name="timeout" value="1" />
  </node>


</launch>
